app/src/main/java/net/gsantner/markor/ui/DraggableScrollbarScrollView.java

```diff
@@ -80,7 +81,8 @@ protected void onAttachedToWindow() {
         if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.JELLY_BEAN_MR1) {
             _ltr = getLayoutDirection() == View.LAYOUT_DIRECTION_LTR;
         }
-        _scrollbarWidth = getVerticalScrollbarWidth();
+        final DisplayMetrics displayMetrics = getContext().getResources().getDisplayMetrics();
+        _grabWidth = (int) (2.0 * (float) getVerticalScrollbarWidth() * displayMetrics.density);
```

Comment:
@gsantner It would be good to test this on other devices

Reply:
I have tested this in the emulator and it seems ok

Reply:
working here as well